mysql-test/mysql-test-run.pl

```diff
@@ -3385,7 +3385,7 @@ ($)
     # to be able to distinguish them from manually created
     # version-controlled results, and to ignore them in git.
     my $dest = "$base_file$suites.result~";
-    my @cmd = ($exe_patch, qw/--binary -r - -f -s -o/,
+    my @cmd = ($exe_patch, qw/-r - -f -s -o/,
```

Comment:
Great, thanks! As simple as it could possibly be and without any OS-specific conditions